ts/parts/Globals.ts

```diff
@@ -87,15 +87,19 @@ declare global {
         ): TReturn;
     }
     interface Document {
+        exitFullscreen: () => Promise<void>;
+        mozCancelFullScreen: Function;
+        msExitFullscreen: Function;
         msHidden: boolean;
+        webkitExitFullscreen: Function;
         webkitHidden: boolean;
     }
     interface Element {
         mozRequestFullScreen: Function;
         msMatchesSelector: Element['matches'];
         msRequestFullscreen: Function;
         webkitMatchesSelector: Element['matches'];
-        webkitRequestFullscreen: Function;
+        webkitRequestFullScreen: Function;
```

Comment:
Please add the `/** @deprecated */` doclet in front of each vendor-specific implementation.